src/views/partials/header.njk

```diff
@@ -1,9 +1,15 @@
-<div class="max-w-7xl mx-auto px-6 py-2">
+<div class="page-container py-3">
     <img src="/assets/images/ons-logo.svg" alt="Office for National Statistics">
 </div>
 
-<header class="mb-4 bg-night-blue text-white">
-    <div class="max-w-7xl mx-auto">
-        <div class="py-4 px-6 text-lg font-bold">Digital Publishing Design System</div>
+<header class="bg-ocean-blue text-white">
+    <div class="page-container flex items-center justify-between py-3">
+        <a href="/" class="sm:text-lg font-bold leading-5 no-underline text-white hover:text-white">Digital Publishing Design System</a>
+        <button id="btn-menu" class="ml-9 py-2 pl-2 pr-1 font-bold text-sm md:hidden border-white border flex items-center relative active:top-px" aria-expanded="false">
+            <span class="uppercase">Menu</span>
+            <svg id="btn-menu-arrow" class="transform" width="20" height="20" viewBox="0 0 20 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
```

Comment:
Might be worth replacing chevron-down.svg with this and then referencing that here instead
(can wrap it around an `<object>` tag)

Reply:
Yeah this is a good point - I forgot to finish this but off but ideally it would be the same sag but also take advantage of `currentColor`. Will see what I can do.

Reply:
Decided to keep them inline for now, can't use `currentColor` with an `<object>` and so we would need 2 versions (white/grey) otherwise.